modules/local/create_pedigree_file.nf

```diff
@@ -0,0 +1,36 @@
+process CREATE_PEDIGREE_FILE {
+    tag "pedigree"
+    label 'process_single'
+
+    input:
+    val meta
+
+    output:
+    path ("*.ped"), emit: ped
+    path "versions.yml", emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    outfile_text = ['#family_id', 'sample_id', 'father', 'mother', 'sex', 'phenotype'].join('\\t')
+    outfile_text += "\\n" + [meta.case, meta.sample, '0', '0', meta.sex, '0'].join('\\t')
```

Comment:
There is no mother or father info in the samplesheet (and it isn't needed to get the sex check)

Reply:
Replaced with the create pedigree from the raredisease pipeline